doc/installation.rst

```diff
@@ -1,23 +1,48 @@
 Installation
 ============
 
+.. _requirements:
+
+Requirements
+------------
+The random forest used in SMAC depends on a c++11 compatible compiler
+and on SWIG (>3.0).
+
+To install both requirements system-wide on a linux system with apt, 
+please call:
+
+.. code-block:: bash
+
+    sudo apt-get install gcc swig
```

Comment:
the reason was that it worked nicely---at least on Ubuntu and Debian. Does `build-essentials` and `gcc` have the same dependencies? Or does `build-essentials` installs more than `gcc`? If yes, is it required in our case?